spec/controllers/followings_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe FollowingsController do
+  before(:each) do
+    request.env["HTTP_REFERER"] = "previous_page"
+  end
+  describe "GET people" do
+    context "with authenticated users" do
+      let(:following) { Fabricate(:following, follower_id: current_user.id) }
```

Comment:
So the before was to satisfy the sessions[:return_to] feature of the app. It would give me an error since there was not previous page to reference. Do you think there is a better way to implement this or just add the before for each describe scenario? Thanks!